backend/private-graph/graph/resolver.go

```diff
@@ -2105,6 +2123,49 @@ func (r *Resolver) AddSlackToWorkspace(ctx context.Context, workspace *model.Wor
 	return nil
 }
 
+func (r *Resolver) RemoveMicrosoftTeamsFromWorkspace(workspace *model.Workspace, projectID int) error {
+
+	if err := r.DB.Transaction(func(tx *gorm.DB) error {
```

Comment:
We remove the tenant integration and the channels saved on the workspace.
We then remove teams channels to notify on all alerts.